### PR TITLE
Nginx checker setup

### PR DESCRIPTION
**Pull request by Fadiabb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,9 @@ config.ini :
 action=create
 u=http://localhost/cgi-bin/csaf_provider.go
 ```
-
+## csaf_checker
+csaf_checker is a tool for testing the CSAF-standards on the csaf_providers.
```
Comment (tschmidtb51) [inline README.md:66]: The csaf_checker (which probably should be named provider checker - at least in the human readable doc when describing its purpose) should not only work with this specific implementation `csaf_provider` but with any CSAF provider
```suggestion
csaf_checker is a tool for testing a CSAF trusted provider according to [Section 7 of the CSAF standard](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#7-distributing-csaf-documents).

File: docs/checker-setup.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Setup checker
+
+The checker deals only with domains and TLS certifcate.
```
Comment (tschmidtb51) [inline docs/checker-setup.md:3]: This seems confusing to me. Doesn't the provider checker check all requirements?

File: docs/checker-setup.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    Create following nginx configuration snippet files:
+    `/etc/nginx/snippets/self-signed.conf` with the content:
+
+    ```bash
+     ssl_certificate /etc/ssl/certs/nginx-selfsigned.crt;
+     ssl_certificate_key /etc/ssl/private/nginx-selfsigned.key;
+    ```
+    and `/etc/nginx/snippets/ssl-params.conf` with the content:
+    ```
+    ssl_protocols TLSv1.2;
```
Comment (tschmidtb51) [inline docs/checker-setup.md:34]: Shouldn't we suggest TLSv1.3 here?

**Review by tschmidtb51 — COMMENTED** [review]
Just a few comments from my side.

**Review by sthagen — APPROVED** [review]
LGTM.

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by Fadiabb** [comment]
@bernhardreiter  yes, i did. I've remove this section with the last commit.

**Pull request merged**